### DISCUSSION TITLE
Is Vite Module Federation plugin ready for real production software?

### DISCUSSION THREAD

**Discussion by joaopedrorc** [webpack/webpack#17352] [Q&A]
What do you think about using Module Federation with Vite via the @originjs/vite-plugin-federation plugin is a suitable approach for migrating a large enterprise application running on ReactJS to micro frontends? I have some reservations about using Vite in this particular scenario.

I'm seeking insights and recommendations on whether this combination of technologies would be a prudent choice for our project. Given the scale and complexity of our application, I have reservations about introducing Vite into the equation. I would greatly appreciate any expert advice or experiences from those who have worked with similar setups or have in-depth knowledge of these tools.

I've tried to deploy to Amazon an example using Webpack and got into a lot of configs as to dev and production and didn't find that much support from Vite plugin.

**Comment by alexander-akait** [comment]
Sorry, I think better to ask this in `@originjs/vite-plugin-federation`.

> I've tried to deploy to Amazon an example using Webpack and got into a lot of configs as to dev and production

Can you describe which?
